content/_index/contact.md

```diff
@@ -0,0 +1,51 @@
++++
+fragment = "contact"
+#disabled = true
+date = "2017-09-10"
+weight = 1100
+#background = "light"
+form_name = "defaultContact"
+
+title = "Contact fragment"
+subtitle  = "*not working on demo page*"
+
+# PostURL can be used with backends such as mailout from caddy
+post_url = "https://example.com/mailout" #default: formspree.io
+email = "mail@example.com"
+button = "Send Button" # defaults to theme default
+#netlify = false
+
+# Optional google captcha
+#[recaptcha]
+#  sitekey = ""
+
+[message]
+  success = "Thank you for awesomely contacting us." # defaults to theme default
+  error = "Message could not be send. Please contact us at mail@example.com instead." # defaults to theme default
+
+# Only defined fields are shown in contact form
+[fields.name]
+  text = "Your Name *"
+  error = "Please enter your name" # defaults to theme default
```

Comment:
All errors default to the theme default. So I would probably go with empty "" for now. Makes it easier for us to iterate on them.

Reply:
text is necessary for now. Maybe we should make that some default to so a field would be enabled by:
```
[fields.name]
```
or
```
[fields.name]
  text = ""
  error = ""
```

Reply:
Text is a required variable in fields. I emptied the error variable and commented it out for good measure.

Reply:
Yeah currently it's the minimum required as we are checking for it. Any reason, why we couldn't check for the field itself? This would make the config simpler in my eyes.

Reply:
But nothing needed here. Either we'll fix it within Syna or push it into the editor fragment.